packages/orchestrator-ui-components/src/components/WFOSteps/WFOStepList/WFOStepList.tsx

```diff
@@ -104,6 +131,32 @@ export const WFOStepList = ({ steps = [], startedAt }: WFOStepListProps) => {
             </EuiFlexGroup>
             <>
                 {steps.map((step, index) => {
+                    let prevState = {};
+                    let delta = {};
+
+                    if (index > 0) {
+                        // Todo: decided if we want this here.
+                        //  V1 has a rather big case statement with businness logic here
+                        if (steps[index].status === 'success') {
```

Comment:
There is a enum with statusses somewhere

Reply:
`step[index]` is the same as `step` in this case